app/views/contests/show.html.erb

```diff
@@ -0,0 +1,71 @@
+<div class="container" data-controller="contest" data-contest-timeleft-value="<%= @contest.deadline %>">
+  <% if notice %>
+    <div class="row" id='alertdiv'>
+      <div class="alert alert-success alert-dismissible container text-center">
+        <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+        <h6><%= notice %></h6>
+      </div>
+    </div>
+  <% end %>
+  <div class="pt-4 d-flex justify-content-between align-items-center">
+    <div>
+      <!-- Changed from translation key to literal "Contest" -->
+      <h3 class="contest-header-title">Contest #<%= @contest.id %></h3>
+      <p class="mb-0 contest-header-description"><%= t("contest.show.subtitle", user_count: @user_count) %></p>
+      <% if @contest.live? %>
+        <p class="mb-0 contest-header-description"><%= t("contest.show.remaining_votes") %> <%= 3 - current_user.user_contest_votes(@contest.id) %></p>
+      <% end %>
```

Comment:
```suggestion
      <% if @contest.live? && current_user.present? %>
        <p class="mb-0 contest-header-description"><%= t("contest.show.remaining_votes") %> <%= 3 - current_user.user_contest_votes(@contest.id) %></p>
      <% end %>
```

If the user is signed out, and visits a contest, it will break the page.